tonic-web/src/service.rs

```diff
@@ -113,7 +116,7 @@ where
                 debug!(kind = "other h2", content_type = ?req.headers().get(header::CONTENT_TYPE));
                 ResponseFuture {
                     case: Case::Other {
-                        future: self.inner.call(req),
+                        future: self.inner.call(req.map(tonic::body::boxed)),
```

Comment:
Hmm, if we have to box the request no matter, what are we gaining with this change?

Reply:
The benefit is the `CorsGrpcWeb` can handle any kind of body types which has the trait bounds.

Reply:
Fair enough. 👍🏻